source/tests/SadWalkerTest.kt

```diff
@@ -0,0 +1,13 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class SadWalkerTest {
+    @Test
+    fun `Should return string with exact same speed and name of walker specified in SadWalker class`() {
+        val speed = 15
+
+        val actual = SadWalker(speed)
```

Comment:
Actual must be final. But u'r use in the assert actual.walk
It's bad practice
Move calling method to actual initialisation